threatconnect.js

```diff
@@ -749,6 +793,44 @@ class ThreatConnect {
     this.request(requestOptions, cb);
   }
 
+  _getDnsInformation(indicatorType, indicatorValue, owner, cb) {
+    let qs = '';
```

Comment:
This will work, but I would recommend using a ternary as it seems like it would be easier to read and avoid mutation.

```
let qs = typeof owner === 'string' && owner.length > 0) ? `
  ?${querystring.stringify({ owner: owner })}`
  : '?includeAdditional=true&includeTags=true'
```